public/pages/LogTypes/containers/CreateLogType.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiLink } from '@elastic/eui';
+import { ContentPanel } from '../../../components/ContentPanel';
+import React, { useContext, useState } from 'react';
+import { LogTypeForm } from '../components/LogTypeForm';
+import { LogTypeBase } from '../../../../types';
+import { defaultLogType } from '../utils/constants';
+import { RouteComponentProps } from 'react-router-dom';
+import { BREADCRUMBS, ROUTES } from '../../../utils/constants';
+import { CoreServicesContext } from '../../../components/core_services';
+import { useEffect } from 'react';
+import { DataStore } from '../../../store/DataStore';
+import { successNotificationToast } from '../../../utils/helpers';
+import { NotificationsStart } from 'opensearch-dashboards/public';
+
+export interface CreateLogTypeProps extends RouteComponentProps {
+  notifications: NotificationsStart;
+}
+
+export const CreateLogType: React.FC<CreateLogTypeProps> = ({ history, notifications }) => {
+  const [logTypeDetails, setLogTypeDetails] = useState<LogTypeBase>({ ...defaultLogType });
+  const context = useContext(CoreServicesContext);
+
+  useEffect(() => {
+    context?.chrome.setBreadcrumbs([
+      BREADCRUMBS.SECURITY_ANALYTICS,
+      BREADCRUMBS.DETECTORS,
+      BREADCRUMBS.LOG_TYPES,
+      BREADCRUMBS.LOG_TYPE_CREATE,
+    ]);
+  }, []);
+
+  return (
+    <ContentPanel
+      title={'Create log type'}
+      subTitleText={
+        <p>
+          Create log type to categorize and identify detection rules for your data sources. &nbsp;{' '}
```

Comment:
Not blocking: 
Should this say "Create **a** log type..."?

Reply:
This one is correct since we are always only creating one log type. Similar to how we say `Create detector`, `Create rule`, etc.